css/style.css

```diff
@@ -902,6 +910,12 @@ button.button-inline:hover {
 	}
 }
 
+@media (max-width: 768px) {
+	#app-navigation-toggle {
+		display: none !important;
```

Comment:
Yes let's add it to mobile as well, since otherwise it is not possible to access archived boards there.